package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "cappadonna",
-  "version": "0.0.0-development",
+  "name": "@soldair/cappadonna",
+  "version": "1.3.1",
```

Comment:
This needs to stay the way it was because semantic-release is responsible for setting the version number before publishing.